pandas/tseries/tests/test_tslib.py

```diff
@@ -177,6 +177,29 @@ def test_constructor_invalid(self):
         with tm.assertRaisesRegexp(ValueError, 'Cannot convert Period'):
             Timestamp(Period('1000-01-01'))
 
+    def test_constructor_positional(self):
+        with tm.assertRaises(TypeError):
+            Timestamp(2000, 1)
+        with tm.assertRaises(ValueError):
```

Comment:
not sure if its worth it to try to make ``Timestamp(2000, 1)`` raise a ``ValueError``